src/App.js

```diff
@@ -36,11 +36,21 @@ import { mapSizesToProps } from './utils/withSizes'
 import styles from './App.module.scss'
 import './App.scss'
 
-const PATHS = {
-  FEED: '/feed'
+export const PATHS = {
+  FEED: '/feed',
+  LOGIN: '/login',
+  LOGIN_VIA_EMAIL: '/login/email',
+  CREATE_ACCOUNT_FREE_TRIAL: '/login/trial',
```

Comment:
@GarageInc /signup intead of trial for path. Cause actually this page is created for signup or create account action